ada_feeding_perception/launch/ada_feeding_perception_launch.xml

```diff
@@ -1,7 +1,7 @@
 <launch>
-  <node pkg="ada_feeding_perception" exec="segment_from_point" name="segment_from_point">
-    <param from="$(find-pkg-share ada_feeding_perception)/config/segment_from_point.yaml"/>
+  <!-- Launch face_detection -->
```

Comment:
I don't like the idea of pushing a change where we are removing food detection. I think it was fine for testing face detection, but we should identify the issue before merging the changes in, so that face detection and food detection can both run at the same time.